apps/web/pages/tasks.tsx

```diff
@@ -35,8 +35,10 @@ export default function Tasks(props) {
 
       <AddTaskItem addTaskItem={addTaskItem} isLoading={isLoading} />
       <VStack
+        h="420px"
+        overflowY="scroll"
         alignItems="flex-start"
-        width={"600px"}>
+        width={"610px"}>
```

Comment:
Pixels are static units. Won't this affect this responsiveness of the site?

Reply:
ooh i was just testing the width feature and forgot to set it back, ill work on that no worries

Reply:
These resources will help you understand CSS units better
- [15 CSS Relative units](https://dev.to/bytegasm/15-css-relative-units-how-many-do-you-know-em-rem-ex-cap-ch-ic-6m)
- [CSS values and units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units)